### PR TITLE
Feature/rbsfxirjtp 202 orderbook service component test

### PR DESCRIPTION
**Pull request by valto2**

### CODE REVIEW

File: order-book/pom.xml [hunk 1]
```diff
@@ -39,6 +39,41 @@
...
+        <dependency>
+            <groupId>org.codehaus.groovy</groupId>
+            <artifactId>groovy</artifactId>
+        </dependency>
+
+        <!-- Grpc-Test-Support -->
+        <dependency>
+            <groupId>org.grpcmock</groupId>
+            <artifactId>grpcmock-junit5</artifactId>
+            <version>0.5.2</version>
```
Comment (zaraandreevaa) [inline order-book/pom.xml:71]: please extract the version in properties

File: order-book/src/test/java/com/market/banica/order/book/service/grpc/componentTests/OrderBookComponentIT.java [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.verify;
+
+@SpringJUnitConfig
+@SpringBootTest(classes = OrderBookApplication.class, webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@RunWith(JUnit4.class)
```
Comment (zaraandreevaa) [inline order-book/src/test/java/com/market/banica/order/book/service/grpc/componentTests/OrderBookComponentIT.java:55]: could you please check if other services are using JUnit5 (I think Radoslav was working on this) and discuss with the team which version you are going to use and have all tests implemented consistently

File: order-book/src/test/java/com/market/banica/order/book/service/grpc/componentTests/OrderBookComponentIT.java [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+
+    @Test
+    void getOrderBookItemLayers_Should_ReturnAUnaryResponse() throws IOException {
+        // Arrange
+        ItemOrderBookResponse expected = generateItemOrderBookExpectedResponse();
+
+        grpcCleanup.register(InProcessServerBuilder
+                .forName(serverName).directExecutor().addService(testConfiguration.getGrpcOrderBookServiceItemLayers(expected))
+                .build().start());
+        // Act
```
Comment (zaraandreevaa) [inline order-book/src/test/java/com/market/banica/order/book/service/grpc/componentTests/OrderBookComponentIT.java:118]: it is a small thing but it is always better to have an empty line before the comments, looks better and is easier for reading

Comment (valto2) [reply order-book/src/test/java/com/market/banica/order/book/service/grpc/componentTests/OrderBookComponentIT.java:118]: Done

File: order-book/src/test/java/com/market/banica/order/book/service/grpc/componentTests/OrderBookComponentIT.java (hunk 2 above)
Comment (valto2) [reply order-book/src/test/java/com/market/banica/order/book/service/grpc/componentTests/OrderBookComponentIT.java:55]: Fixed

File: order-book/pom.xml (hunk 1 above)
Comment (valto2) [reply order-book/pom.xml:71]: Done

**Review by gaantonova — APPROVED** [review]

**Review by radevdetelin — APPROVED** [review]

File: order-book/src/test/java/resources/application-testOrderBookIT.properties [hunk 4]
```diff
@@ -0,0 +1,4 @@
+product-name=cheese
```
[the remark this answers is not in the document]
Comment (valto2) [reply order-book/src/test/java/resources/application-testOrderBookIT.properties]: Fixed

**Pull request merged**